include/parametric-curves/text-file.hpp

```diff
@@ -66,21 +66,27 @@ struct TextFile :
   virtual bool loadTextFile(const std::string& fileName)
   {
     Eigen::MatrixXd data = parametriccurves::utils::readMatrixFromFile(fileName);
-    if(data.cols()!=3*size)
+    if(data.cols()==size)
+    {
+      std::cout<<"Setting derivatives to zero"<<std::endl;
+      posValues = data;
+      velValues.setZero();
+      accValues.setZero();
```

Comment:
I feel this might cause problems if `velValues` and `accValues` do not have the right dimension.
```
velValues.setZero(size)
accValues.setZero(size)
```
is safer in my opinion

Reply:
It would certainly cause problems if `velValues` is read. On the other hand, if this option is triggered, you *should not* read velValues, so maybe we *want* it to crash?